src/app/shared-component/dialog/dialog.component.ts

```diff
@@ -39,4 +40,22 @@ export class DialogComponent implements OnInit {
     this.onClick.emit(val);
     this.modalFadeIn = false;
   }
+
+  @HostListener('window:keydown', ['$event'])
+  onActionKeyPress(event: any): void {
+    let currentDefaultIndex = this.dialog.actionButtons.findIndex(i => i.default);
```

Comment:
What is "default" here? I'm getting a Type error on it each time it is used in this method.
Error:(79, 74) TS2339:Property 'default' does not exist on type 'Object'.